test/data/filters/Latex/file-latex-items-exp.tex

```diff
@@ -0,0 +1,10 @@
+\documentclass{article} \begin{document} \title{LaTeX Itemize example} 
```

Comment:
You can find the latex filter does not produce same text with same line breaks.
A enhance here does not and can not change the behavior of line-breaks.